runtime/altair/src/migrations.rs

```diff
@@ -28,6 +28,7 @@ frame_support::parameter_types! {
 
 /// The migration set for Altair 1034 @ Kusama. It includes all the migrations
 /// that have to be applied on that chain.
+#[cfg(not(feature = "std"))]
```

Comment:
We should remove this now (all green lines from `migrations.rs`). Integration test no longer migrates anything